packages/app/src/runs/modals/CreateCloudOrgModal.vue

```diff
@@ -101,29 +94,20 @@ const props = defineProps<{
 }>()
 
 const query = useQuery({
-  query: CheckCloudOrganizationsDocument,
+  query: CloudOrganizationsCheckDocument,
   requestPolicy: 'network-only',
+  pause: true,
```

Comment:
Why is this `pause: true`? Is that because we want to trigger it manually, but not automatically?